src/domain/orders/draft-orders/details.tsx

```diff
@@ -136,6 +163,31 @@ const DraftOrderDetails = ({ id }) => {
     })
   }
 
+  const handleAddDiscount = (data: AddDiscountCodeInput) => {
+    const discount = discounts?.find(
```

Comment:
where does `discounts` come from? I can't imagine that we're pulling EVERY discount for this page right? We could have hundreds.

Reply:
`discounts` is coming from up above. i called `useAdminDiscounts()` to grab all of the available discounts and map them into options for the select input, so the user can make their choice. we don't need to do that here, so this particular instance is resolved, but i do still wonder if there aren't params we can send thru as options to narrow down the size of the returned discounts. what do you think?

Reply:
Hmmm, yea you can add `is_dynamic=false` as a query param and that will help with not showing the generated discounts.

Reply:
this is resolved via the new search box!